Events/FoldUnfoldImplementEvent.lua

```diff
@@ -0,0 +1,53 @@
+FoldUnfoldImplementEvent = {};
```

Comment:
Are you sure we need this? There's already a standard Giants FoldableSetFoldDirectionEvent, isn't that triggered when setFoldDirection() is called?

Reply:
Your right, I didn't quite catch that one. Seems like the Folding bugs are related to "onAiStart"